spec/lib/appsignal_spec.rb

```diff
@@ -427,17 +427,17 @@
     describe "custom stats" do
       describe ".set_gauge" do
         it "should call set_gauge on the extension with a string key and float" do
-          expect(Appsignal::Extension).to receive(:set_gauge).with("key", 0.1)
+          expect(Appsignal::Extension).to receive(:set_gauge).with("key", 0.1, Appsignal::Extension.data_map_new)
           Appsignal.set_gauge("key", 0.1)
```

Comment:
This doesn't really test if the value from the tags gets passed to the extension.
Can you add a spec for when there's a tags value and where there isn't?